src/test/java/io/confluent/connect/jdbc/sink/DbStructureTest.java

```diff
@@ -41,6 +60,110 @@ public void testMissingFieldsWithDifferentCase() {
     assertTrue(missingFields(sinkRecords("AaA", "bBb"), columns("aAa", "BbB")).isEmpty());
   }
 
+  @Test (expected = TableAlterOrCreateException.class)
+  public void testMissingTableNoAutoCreate() throws Exception {
+    structure.create(config, connection, tableId,
+        fieldsMetadata);
+  }
+
+  @Test (expected = TableAlterOrCreateException.class)
+  public void testAlterNoAutoEvolve() throws Exception {
+    TableDefinition tableDefinition = mock(TableDefinition.class);
+    when(dbDialect.tableExists(any(), any())).thenReturn(true);
+    when(dbDialect.describeTable(any(), any())).thenReturn(tableDefinition);
+    when(tableDefinition.type()).thenReturn(TableType.VIEW);
```

Comment:
Won't we technically fail here because the table is a view, not because auto-evolve is disabled in the connector config?

Reply:
You're right, fixed.